.github/workflows/build_android_library.yml

```diff
@@ -0,0 +1,17 @@
+# Builds the TFLite Java Bindings Android Library using the gradle-android
+# Docker image.
+name: Build Android Library
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build_android_with_docker:
+    runs-on: ubuntu-latest
+    container: gcr.io/iree-oss/gradle-android@sha256:117aef7fc63facb92003cef6e5869c9759bce44ca51e5152ebc68cca253920ee
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+    - name: Execute Android Build
+      run: build_tools/gradle/build_android.sh
```

Comment:
The eventual plan is to move this into https://github.com/google/iree/blob/main/.github/workflows/build_package.yml, right?

Could rename this workflow to be more like https://github.com/google/iree/blob/main/.github/workflows/macos_oneshot_build.yml (`android_tflite_oneshot_build.yml`?)

Reply:
The end goal is to tie it into the build_package.yml. I'll leave it up to @not-jenni  to decide if she wants to run this as a sub build or just copy the action (its small enough) into that file. 

 I'm uploading this as a standalone workflow first for reviewing/testing purposes. Will rename.